include/secp256k1_bulletproofs.h

```diff
@@ -0,0 +1,144 @@
+#ifndef _SECP256K1_BULLETPROOFS_
+# define _SECP256K1_BULLETPROOFS_
+
+# include "secp256k1.h"
+# include "secp256k1_generator.h"
+
+# ifdef __cplusplus
+extern "C" {
+# endif
+
+#include <stdint.h>
+
+/** A function-like macro returning the size, in bytes, of an uncompressed
+ * Bulletproof proving that a value lies in the range [0, 2^(n_bits) - 1]
+ */
+#define SECP256K1_BULLETPROOFS_UNCOMPRESSED_SIZE(n_bits) (194UL + (n_bits) * 64)
```

Comment:
Comment: Is it worth mentioning where 194 came from (65 for A, S, 65 for T1, T2, 64 for tau_x, mu)?

Reply:
I'll leave it be, because this commit isn't mine, and I don't think it's a huge deal, and also this'll change with BP++.

Reply:
Ah, I have to edit that commit anyway to address the above comment. Ok.